examples/migrate_forseti/tutorial.md

```diff
@@ -0,0 +1,362 @@
+  startLine=51
+  endLine=51
+  startCharacterOffset=21
+  endCharacterOffset=28>subnetwork</walkthrough-editor-select-line>.
+
+
+### Shared VPC
+If using a shared VPC, you'll need to update the related input variables.
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="SHARED_VPC_PROJECT_ID">network_project</walkthrough-editor-select-regex>
+to match the project in which your shared VPC exists.
+
+### Update Region
+By default, Forseti deploys into us-central1.  If you deployed Forseti in a different region, you will need to
+update the cloudsql_region, server_region, and client_region input variables.
+
+Update the <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="us-central1">cloudsql_region</walkthrough-editor-select-regex>
+to match the region where CloudSQL is deployed.
+
+Update the <walkthrough-editor-select-line
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  startLine=63
+  endLine=63
+  startCharacterOffset=21
+  endCharacterOffset=33>server_region</walkthrough-editor-select-line>
+to match the region where the Forseti Server VM is deployed.
+
+Update the <walkthrough-editor-select-line
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  startLine=64
+  endLine=64
+  startCharacterOffset=21
+  endCharacterOffset=33>client_region</walkthrough-editor-select-line>
+to match the region where the Forseti Client VM is deployed.
+
+## Add Input Variables for Custom Configurations
+Starting with Forseti Security 2.23, Terraform will manage your server
+ configuration file for you.  Configuration options will now be input
+ variables that are defined in the Terraform module.  This will ensure
+ upgrading Forseti will be as easy as possible going forward.
+
+Please identify any Forseti server configuration variables that have
+been customized and add them to your <walkthrough-editor-open-file filePath="terraform-google-forseti/examples/migrate_forseti/main.tf">main.tf</walkthrough-editor-open-file>.
+
+For example, if you modified the e-mail connector recipient in the **forseti_conf_server.yaml** file in the Forseti Server GCS bucket:
+```yaml
+email_connector:
+      name: sendgrid
+      auth:
+        api_key: ...
+      sender: ...
+      recipient: myemail@example.com
+      data_format: csv
+```
+You will want to add the
+`forseti_email_recipient = myemail@example.com`
+to your <walkthrough-editor-select-regex
+  filePath="terraform-google-forseti/examples/migrate_forseti/main.tf"
+  regex="Add any Forseti Server Configuration Variables Here">main.tf</walkthrough-editor-select-regex>
+
+## Obtain and Run the Import Script
+This [import script](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/import.sh) will import the Forseti GCP resources into a local state file.
```

Comment:
We should tell users to look at the -h output to populate the needed flags.

Reply:
actually, this step should be moved after the `terraform init` step

Reply:
Moved this as suggested. Added instruction to refer the output of `import.sh -h` command to populate flags as needed.